src/components/StatusChip/StatusChip.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { createThemeModeVariants } from '@ses/core/utils/storybook/factories';
+import { StatusChip } from './StatusChip';
+import type { Meta } from '@storybook/react';
+
+const meta: Meta<typeof StatusChip> = {
+  title: 'Fusion/Components/StatusChip',
+  component: StatusChip,
+
+  parameters: {
+    chromatic: {
+      viewports: [375, 768, 1024, 1280, 1440, 1920],
```

Comment:
This component looks to be the same in all the resolutions so we don't need to specify the `viewports` for this case

Reply:
you are right, done